### ISSUE TITLE
overfitting when training the kpts

### ISSUE THREAD

**Issue by yongguanjiangshan** [TexasInstruments/edgeai-yolov5#110]
## ❔Question
Hi，when i train the model with below script, the val/box_loss and val/obj_loss are increasing. And the  train/cls_loss is always 0. As the piture show，do you know how to solve this problem?

> python train.py --data coco_kpts.yaml --cfg yolov5s6_kpts_ti_lite.yaml --weights ./pretrained_weights/Yolov5s6_pose_960_ti_lite/last.pt --batch-size 64 --img 960 --kpt-label --hyp hyp.scratch_lite.yaml

## Additional context
![image](https://user-images.githubusercontent.com/18051804/231047336-36b6d895-219e-4d7e-8d3c-928742da48f6.png)

**Comment by clw5180** [comment]
cvmart?

**Comment by yongguanjiangshan** [comment]
> How many samples in your training and validation dataset? Which dataset do you use ?

I use coco. I have found the bug in train.py.  change the below code
`tags = ['train/box_loss', 'train/obj_loss', 'train/cls_loss',  # train loss
                    'metrics/precision', 'metrics/recall', 'metrics/mAP_0.5', 'metrics/mAP_0.5:0.95',
                    'val/box_loss', 'val/obj_loss', 'val/cls_loss',  # val loss
                    'x/lr0', 'x/lr1', 'x/lr2']  # params`
to
`tags = ['train/box_loss', 'train/obj_loss', 'train/cls_loss',  # train loss
                    'train/kpt', 'train/kptv', 'metrics/precision', 'metrics/recall',
                    'metrics/mAP_0.5', 'metrics/mAP_0.5:0.95', 'val/box_loss',  # val loss
                    'x/lr0', 'x/lr1', 'x/lr2']  # params`
then the tensorboard log will be normal.

**Comment by yongguanjiangshan** [comment]
> How many samples in your training and validation dataset? Which dataset do you use ?

Hi, I use coco2017. I have found the bug in train.py. just convert the below code 
`tags = ['train/box_loss', 'train/obj_loss', 'train/cls_loss',  # train loss
                    'metrics/precision', 'metrics/recall', 'metrics/mAP_0.5', 'metrics/mAP_0.5:0.95',
                    'val/box_loss', 'val/obj_loss', 'val/cls_loss',  # val loss
                    'x/lr0', 'x/lr1', 'x/lr2']  # params`
to 
`tags = ['train/box_loss', 'train/obj_loss', 'train/cls_loss',  # train loss
                    'train/kpt', 'train/kptv', 'metrics/precision', 'metrics/recall',
                    'metrics/mAP_0.5', 'metrics/mAP_0.5:0.95', 'val/box_loss',  # val loss
                    'x/lr0', 'x/lr1', 'x/lr2']  # params`
and train with the script

> python train.py --data coco_kpts.yaml --cfg yolov5s6_kpts_ti_lite.yaml --weights '' --batch-size 64 --img 640 --kpt-label --hyp hyp.scratch_lite.yaml

then the tensorboard logs will be normal.
